### ISSUE TITLE
Can't build my Stencil project because of incorrect import

### ISSUE THREAD

**Issue by laddi-netapp** [break-stuff/wc-storybook-helpers#6]
Hi, I'm using Stencil with Storybook and wanted to see if I could use your package to simplify my story writing.  After following the installation instructions provided I'm faced with this:

```
[ ERROR ]  TypeScript: node_modules/wc-storybook-helpers/dist/storybook-utils.d.ts:2:40
           Cannot find module './storybook' or its corresponding type declarations.

      L1:  import { TemplateResult } from "lit";
      L2:  import type { ArgTypes, Options } from "./storybook";
      L3:  /**

[31:31.6]  build failed in 2.32 s
```

The `dist` folder doesn't contain a `storybook.*` file so I'm wondering if there might be an issue in the bundling or publishing on your end?

**Comment by break-stuff** [comment]
It should resolve to your installed instance. What version of storybook do you have installed?

**Comment by break-stuff** [comment]
Also, is your set-up somewhere I can take a look at?

**Comment by break-stuff** [comment]
Ah, sorry. I see the problem. Let me see if I can get that resolved.

**Comment by break-stuff** [comment]
The fix was resolved in #8 and you should be able to get the fix in 1.0.3. Let me know if anything else comes up. Thanks!
